### ISSUE TITLE
Script terminates when reporting against an empty datastore cluster

### ISSUE THREAD

**Issue by tpcarman** [AsBuiltReport/AsBuiltReport.VMware.vSphere#35]
**Describe the bug**
When reporting against a VMware vSphere environment which contains an empty datastore cluster, the script terminates with a `Attempted to divide by zero` error.

**Comment by tpcarman** [comment]
Issue identified to be with the calculation of `% Used`. Will implement the following code changes to resolve the issue.

```
'% Used' = Switch ($DSCluster.CapacityGB -gt 0) {
    $true { [math]::Round((100 - (($DSCluster.FreeSpaceGB) / ($DSCluster.CapacityGB) * 100)), 2) }
    $false { '0' }
}
```

**Comment by tpcarman** [comment]
Resolved in #40
